synapse/lib/types.py

```diff
@@ -710,8 +710,9 @@ def norm(self, valu):
             raise s_exc.BadTypeValu(mesg=mesg)
 
         if self.opts.get('norm'):
```

Comment:
Have you thought at all about not allowing floats?

Reply:
Yeah I don't think we'll be able to allow floats if we're going to limit the number of decimal places.

Reply:
Since we're rounding maybe theyre okay?

Reply:
Yeah seems like it.